sirepo/template/template_common.py

```diff
@@ -270,23 +270,36 @@ def compute_plot_color_and_range(plots, plot_colors=None, fixed_y_range=None):
     return y_range
 
 
-def dict_to_h5(d, hf, path=None):
-    if path is None:
-        path = ''
+def write_dict_to_h5(d, file_path, h5_path=None):
+    """ Store the contents of a dict in an h5 file starting at the provided path.
+    Stores the data recursively so that
```

Comment:
In order for this to work, you need to have a `::` so that napoleon-sphinx picks it up as a code block.